lib/swaps/Swaps.ts

```diff
@@ -56,7 +53,7 @@ interface Swaps {
 }
 
 class Swaps extends EventEmitter {
-  /** A mapping containing two entries for each deal, one using dealId as a key and one using r_hash as a key. */
+  /** A mapping containing an entry for each deal, using r_hash as a key. */
```

Comment:
Now that there's only one key used, this can just say "A map between r_hash and swap deals."